### DISCUSSION TITLE
Use arbitrary Stream as source

### DISCUSSION THREAD

**Discussion by TopperDEL** [CommunityToolkit/Maui#975] [New Feature Discussions]
I have an app where I need to use an arbitrary Stream to provide the data for a Video Playback with the new MediaElement. Can that be supported?

As I'm loading the Video directly from a decentralized cloud storage I have no intermittend Service with a "normal" URI. I fetch the data with a library that Providers a Stream.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
